### DISCUSSION TITLE
Bienvenu dans la communauté Montréal en commun

### DISCUSSION THREAD

**Discussion by houssamkhalifeh** [VilledeMontreal/MontrealEnCommun#7] [Announcements]
# 👋 Bienvenu
La communauté Montréal en commun utilise les *Discussions* comme une place d’échanges entre les membres et participants de la communauté. Utiliser les discussion pour:

- Poser vos questions
- Partager vos idées 💡
- Communiquer avec les membres de la communauté
- Consulter les suggestions
- ..

Pour commencer, SVP ajouter un commentaire avec le nom de votre entreprise, votre rôle et vos suggestions pour une meilleure collaboration.

**Comment by pagbokou-artm** [comment]
Nom: Patrick Agbokou
Entreprise: ARTM
Projet:  Plateforme métropolitaine de mobilité servicielle (Mobility as a Service)

**Comment by dgendron-solon** [comment]
Salut!
Nom : David Gendron
Entreprise : Solon
Projet : locomotion.app

**Comment by vince-fabmob** [comment]
Salut! 
Nom: Vincent Dussault
Entreprise: LIUM
Projet: Montréal en commun (volet mobilité)

**Comment by MagaliCantine** [comment]
Bonjour,
Nom : Magali Morin
Organisation : La Cantine pour tous
Projet : MutuAli - mutualisation d'équipements alimentaires

**Comment by lgharib** [comment]
Nom: Larbi Gharib
Entreprise: FabmobQC
Projets: 
- Ma Mobilité : https://fabmobqc.ca/ma-mobilite/
- Conversion des données ouvertes de stationnement de la ville de Montréal au format CurbLR
